src/EFCore.PG/Extensions/NpgsqlEntityTypeBuilderExtensions.cs

```diff
@@ -109,6 +109,43 @@ public static class NpgsqlEntityTypeBuilderExtensions
 
         #endregion Comment
 
+        #region Unlogged Table
+
+        /// <summary>
+        /// Configures the entity to use an unlogged table when targeting Npgsql.
+        /// </summary>
+        /// <param name="entityTypeBuilder">The builder for the entity type being configured.</param>
+        /// <param name="unlogged">True to configure the entity to use an unlogged table; otherwise, false.</param>
+        /// <returns>
+        /// The same builder instance so that multiple calls can be chained.
+        /// </returns>
+        public static EntityTypeBuilder ForNpgsqlHasUnloggedTable(
```

Comment:
Let's rename this to `ForNpgsqlIsUnlogged()`, which is the convention for boolean/flag properties (e.g. [ForSqlServerIsMemoryOptimized](https://github.com/aspnet/EntityFrameworkCore/blob/master/src/EFCore.SqlServer/Extensions/SqlServerEntityTypeBuilderExtensions.cs#L44)). This would mean renaming all other related methods/properties.